static/js/script.js

```diff
@@ -1,4 +1,43 @@
 
 function onloadPage(){
-    console.log(data);
+    // to load resources on UI as button
+  let html = ``;
+  for (let i = 0; i < data.length; i++) {
+    
+    let resourceName = Object.keys(data[i])[0];
+    html += `<button class="btn bg-white btn-change7 text-left `;
+    html += resourceName;
+    html += `" >`;
+    html += resourceName;
+    html += `</button>`;
+  }
+
+  let addButton = document.getElementById("resources-button");
+  addButton.innerHTML = html;
+
+  // to add onclick listener on resource button
+  for (let i = 0; i < data.length; i++) {
+    let button_Name = "." + Object.keys(data[i])[0];
+    $(document).on("click", button_Name, function () {
+    //   load(endpoint[i], resources[i], i);
+    //   query();
+    driver("Patient");
+    console.log(Object.keys(data[i])[0]);
+    });
+  }
+}
+
+async function driver(selectedResource){
+    resource = selectedResource;
+    let param = data[0]["Patient"]["queryParam"][0].name;
+    let value = data[0]["Patient"]["queryParam"][0]["value"][0];
+    console.log(param);
```

Comment:
I will allow them, as it's in active dev. But I need a PR which removes them at a later stage. @rishi2907 make an open issue for this `removing console.logs`